pkg/vpcmodel/output.go

```diff
@@ -168,14 +179,15 @@ func (of *serialOutputFormatter) createSingleVpcFormatter() SingleVpcOutputForma
 
 func (of *serialOutputFormatter) WriteOutput(c1, c2 map[string]*VPCConfig, conns map[string]*VPCConnectivity,
 	subnetsConns map[string]*VPCsubnetConnectivity, subnetsDiff *diffBetweenCfgs,
-	outFile string, grouping bool, uc OutputUseCase) (string, error) {
+	outFile string, grouping bool, uc OutputUseCase, explainStruct *Explanation) (string, error) {
 	diffAnalysis := uc == EndpointsDiff || uc == SubnetsDiff
-	if !diffAnalysis {
+	if !diffAnalysis && uc != Explain {
```

Comment:
This is a bit ugly; please add a name for `uc == Explain` or, even better, have one name for both diffAnalysis and explainMode e.g.

` singleVPCAnalysis := (uc == EndpointsDiff || uc == SubnetsDiff) || uc == Explain`